EIPS/eip-5218.md

```diff
@@ -0,0 +1,237 @@
+---
+eip: 5218
```

Comment:
?!!??!?!?! Why is this here?

Reply:
This was because branch got outdated, merged with latest 👍